libs/saveFiles.js

```diff
@@ -56,15 +56,18 @@ const downloadEntry = function (entry, folderPath) {
       })
     })
     .then(fileobject => {
+      checkMimeWithPath(fileobject.attributes.mime, fileobject.attributes.name)
+      checkFileSize(fileobject)
+      return fileobject
```

Comment:
Why are the checks here and below ? Since the checks return true/false, I think they do nothing here ?